crates/domain/src/repositories/task_repository.rs

```diff
@@ -29,33 +30,40 @@ impl TaskRepository for DomainTaskRepository {
     async fn get_tasks(&self) -> anyhow::Result<Vec<Task>> {
         let tasks: Vec<Task> = query(
             "
+            WITH tags_array AS (
```

Comment:
What does this do?

Reply:
https://chatgpt.com/share/673aca5f-7d64-8011-b369-0b4e573adff2